packages/sanity/src/core/presence/DocumentPreviewPresence.tsx

```diff
@@ -70,7 +70,7 @@ export function DocumentPreviewPresence(props: DocumentPreviewPresenceProps) {
   return (
     <Tooltip content={tooltipContent} {...PRESENCE_MENU_POPOVER_PROPS}>
       <AvatarStackCard scheme={selected ? invertedScheme : undefined} $selected={selected}>
-        <AvatarStack maxLength={2}>
+        <AvatarStack maxLength={2} aria-label="Who is here">
```

Comment:
Could we use the string returned from `getTooltipText` as aria label?

Reply:
Yes, will update it! Thanks 😊